domains/certificates/RegistryConnector/Worker.UnitTests/RoutingSlips/IssueToRegistryActivityTests.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using Google.Protobuf;
+using MassTransit;
+using Microsoft.Extensions.Logging;
+using NSubstitute;
+using ProjectOrigin.Common.V1;
+using ProjectOrigin.Registry.V1;
+using RegistryConnector.Worker.RoutingSlips;
+using Xunit;
+
+namespace RegistryConnector.Worker.UnitTests.RoutingSlips;
+
+public class IssueToRegistryActivityTests
+{
+    [Fact]
+    public void ShouldIssueToRegistry()
+    {
+        var clientMock = Substitute.For<RegistryService.RegistryServiceClient>();
+        var loggerMock = Substitute.For<ILogger<IssueToRegistryActivity>>();
+        var contextMock = Substitute.For<ExecuteContext<IssueToRegistryArguments>>();
+
+        contextMock.Arguments.Returns(new IssueToRegistryArguments(new Transaction
+        {
+            Header = new TransactionHeader
+            {
+                FederatedStreamId = new FederatedStreamId
+                {
+                    Registry = "SomeRegistry",
+                    StreamId = new Uuid
+                    {
+                        Value = Guid.NewGuid().ToString()
+                    }
+                },
+                Nonce = "Dunno what this is",
```

Comment:
wouldnt a nonce be a random hash value? Maybe we should clarify this :)

Reply:
I've asked Martin